src/pages/Route/typing.d.ts

```diff
@@ -157,5 +157,70 @@ declare namespace RouteModule {
       [name: string]: any;
     };
     script: Record<string, any>;
+    url?: string;
+  };
+
+  // for route debug
+  interface Server {
```

Comment:
please use `type` instead of `interface` to prevent from type override :D

Reply:
done